src/test/groovy/page/LoginPage.groovy

```diff
@@ -10,5 +10,23 @@ class LoginPage extends Page{
     static at = { title == "Zaloguj się - ExamPlanner" }
 
     static content = {
+        header { $(".u-centered").text() }
+        emailLabel { $(".control-label", 0, for: "Email").text() }
+        emailInput { $("#Email") }
+        passwordLabel { $(".control-label", 0, for: "Password").text() }
+        passwordInput { $("#PasswordPass") }
+        passwordMaskToggle { $("#PasswordSwitch") }
+        rememberPasswordCheckbox { $("#RememberMe", 0, type: "checkbox") }
+        rememberPasswordLabel { $("label", 0, for: "RememberMe").text() }
+        remindPasswordButton { $("input", 0, value: "Przypomnij hasło") }
+        logInButton { $("input", 0, value: "Zaloguj się") }
+    }
+
+    static def "log in with email and password" (email, password) {
+        this.emailInput = email
```

Comment:
1. Wydaje mi sie, ze this, jest tuaj nie potrzebne.
2. przypisujesz wartosc do wartoscie selectora, wydaje mi sie, ze to nie bedzie wpisywac nic w pola. Zeby to dzialalo, uzyj ""emailInput << email" albo emailInput.value(email).
3. Lepiej odrazu testowac takie metody, tworzoc jakis prosty test.